integration_tests/pages/manage/placements/nonArrival.ts

```diff
@@ -0,0 +1,48 @@
+import { faker } from '@faker-js/faker'
+import type { Cas1SpaceBooking } from '@approved-premises/api'
+import Page from '../../page'
+import paths from '../../../../server/paths/manage'
+
+export class RecordNonArrivalPage extends Page {
+  constructor(title: string = 'Record someone as not arrived') {
+    super(title)
+  }
+
+  shouldShowReasonError(): void {
+    cy.get('.govuk-error-summary__list').should('contain', 'You must select a reason for non-arrival')
+  }
+
+  shouldShowTextError(): void {
+    cy.get('.govuk-error-summary__list').should('contain', 'You have exceeded 200 characters')
+  }
```

Comment:
The parent class `Page` offers a `shouldShowErrorMessagesForFields` method which means the error messages can be seen directly in the cypress test file, and also tests both the error summary and the error by the field -- e.g.:

```js
// nonArrival.cy.ts

const page = new RecordNonArrivalPage()
// submit the form, etc
page.shouldShowErrorMessagesForFields(['notes'], {
  notes: 'You have exceeded 200 characters'
})
```

As an aside, is there any reason this field has a limit on character count? This seems a bit of an odd outlier.